### PR TITLE
update games/tbftss to 1.3

### PR DESCRIPTION
**Pull request by romeroyakovlev**

### CODE REVIEW

File: games/tbftss/pkg/PLIST [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+share/tbftss/sound/33785__jobro__4-beep-b.ogg
+share/tbftss/sound/39030__wildweasel__d1clsstf.ogg
+share/tbftss/sound/42106__marcuslee__laser-wrath-4.ogg
+share/tbftss/sound/47252__nthompson__bad-explosion.ogg
+share/tbftss/sound/49678__ejfortin__energy-short-sword-7.ogg
+share/tbftss/sound/56246__q-k__latch-04.ogg
+share/tbftss/sound/62491__benboncan__dslr-click.ogg
+share/tbftss/sound/65787__iwilldstroyu__laserrocket.ogg
+share/tbftss/sound/77087__supraliminal__laser-short.ogg
+share/tbftss/sound/88275__s-dij__gbc-reload-06.ogg
```
Comment (sizeofvoid) [inline games/tbftss/pkg/PLIST]: You miss


@exec %D/bin/update-desktop-database
@unexec-delete %D/bin/update-desktop-database
@exec %D/bin/gtk-update-icon-cache -q -t %D/share/icons/hicolor
@unexec-delete %D/bin/gtk-update-icon-cache -q -t %D/share/icons/hicolor


and 

RUN_DEPENDS += devel/desktop-file-utils \
x11/gtk+3,-guic

File: games/tbftss/Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			ICON_DIR=${PREFIX}/share/icons/hicolor \
+			DESKTOP_DIR=${PREFIX}/share/applications \
+			DATA_DIR=${PREFIX}/share/tbftss
+
+USE_GMAKE =		Yes
+
+NO_TEST =		Yes
+
+MAKE_FILE =		makefile
+WRKDIST =		${WRKDIR}/tbftss-$V
```
Comment (sizeofvoid) [inline games/tbftss/Makefile]: WRKDIST =		${WRKDIR}/${PKGNAME}

**Comment by bentley** [comment]
I came up with a similar update about a month ago. I got stuck resolving [this crash on exit](stephenjsweeney/tbftss/issues/27) though. Have you hit the same problem?

**Comment by romeroyakovlev** [comment]
0.4 from official ports also crash on exit

**Comment by bentley** [comment]
Ugh, pretty sure it didn’t back when it was imported. There’s an alarming number of bugs affecting SDL ports lately (this, the SDL joysticks hang, the SDL perl build failure)…

**Comment by ibara** [comment]
Plus like all SDL ports have terrible performance compared to other OSes (a long-term probably I've been meaning to tackle but have not).

**Comment by thfrwn** [comment]
Just wanna add to the discussion that SDL2 ports that I'm working on don't find gamepads (FNA games, fs2open). dxx-rebirth does find flightstick/gamepad, but that one is SDL1.

**Comment by bentley** [comment]
That problem probably isn’t inherent to SDL2—mgba, mupen64plus, and retroarch use SDL2 and I use gamepads with them regularly.

**Comment by thfrwn** [comment]
I'm just debugging this. It's the GameController API - SDL2 reports both a joystick and a gamecontroller device. The joystick is reported correctly with SDL2 and is likely what those emulators look for. Here a pastebin with sdl-jstest that is helping me with debugging:

https://pastebin.com/EZ1EdkCQ

I've worked my way to the function SDL_SYS_JoystickNameForDeviceIndex failing when I try to have it find the gamecontroller the Linux way.

If I hit a brick wall, I'll post how far I got. Otherwise, you'll hear about a patch proposal :)

**Comment by thfrwn** [comment]
I got it working, except for the fact that the y axis of the sticks is inverted and the D-pad of my Xbox 360 controller is still not working. Wanna take a look at these before proposing a patch.

Also SDL2 2.0.5 is a little outdated - they are at 2.0.7 now and apparently support syntax to invert axes starting in 2.0.6... Not sure if I have the bandwidth to tackle the update though...

**Comment by thfrwn** [comment]
If anyone knows if OpenBSD-side we can invert the values returned by Game_Pad.Y, let (can see with `usbhidctl -f /dev/uhid0 -l`).

**Comment by ibara** [comment]
Is there any reason to keep this open? Can someone commit to just making the necessary tweaks?

**Comment by bentley** [comment]
Right, there’s really no reason to have left this so long. Sorry about that.

I’ve committed the update. Thanks!

**Pull request closed without merging**